ssl/ssl_lib.c

```diff
@@ -6540,26 +6661,34 @@ int ssl_log_rsa_client_key_exchange(SSL_CONNECTION *sc,
         return 0;
     }
 
+#ifndef OPENSSL_NO_SSLKEYLOG
     /* We only want the first 8 bytes of the encrypted premaster as a tag. */
     return nss_keylog_int("RSA",
                           sc,
                           encrypted_premaster,
                           8,
                           premaster,
                           premaster_len);
+#else
+    return 0;
+#endif
 }
 
 int ssl_log_secret(SSL_CONNECTION *sc,
                    const char *label,
                    const uint8_t *secret,
                    size_t secret_len)
 {
+#ifndef OPENSSL_NO_SSLKEYLOG
     return nss_keylog_int(label,
                           sc,
                           sc->s3.client_random,
                           SSL3_RANDOM_SIZE,
                           secret,
                           secret_len);
+#else
+    return 0;
```

Comment:
Similarly here. This needs to return 1, otherwise you break `tls_construct_finished()`
and `quic_change_cipher_state()`. Easier to drop the #ifdefery.